sdk/eventhub/event-hubs/samples/javascript/package.json

```diff
@@ -30,7 +30,7 @@
     "@types/dotenv": "^8.2.0",
     "@types/ws": "^6.0.4",
     "dotenv": "^8.2.0",
-    "https-proxy-agent": "^3.0.1",
```

Comment:
@willmtemple, @chradek: I updated the samples to match the runtime package.  Keep the sample changes or revert?

Reply:
I believe the dev dependency to `event-hubs` was added for the sake of samples at a time when samples didnt have their own package.json. So, you should be able to remove dev dependency altogether from `service-bus` and `event-hubs`

Reply:
Will do.  I believe it needs to be kept in `event-processor-host` though because it's samples don't have their own `package.json`.

Reply:
@ramya-rao-a: But you otherwise agree the version should be updated in the samples?

Reply:
We don't have much to lose if it is updated or not. I am ok going either way